leapp/utils/report.py

```diff
@@ -139,4 +139,13 @@ def generate_report_file(messages_to_report, context, path, report_schema='1.1.0
                     groups = msg.pop('groups', [])
                     msg['flags'] = [g for g in groups if g in _DEPRECATION_FLAGS]
                     msg['tags'] = [g for g in groups if g not in _DEPRECATION_FLAGS]
+            if report_schema_tuple == (1, 2, 0):
```

Comment:
Nitpick: imho checking for exact version of report schema might cause problems when we bump the schema version sooner than the flags/tags deprecation period has elapsed. However, I am not sure how often such version change is performed, and therefore, my concerns might not be necessary. 
```suggestion
            if report_schema_tuple >= (1, 2, 0):
```

Reply:
Basically I chose this solution as we do not have any idea what will be in another version. We know only what's in the current one and previous versions. With the bump of the schema, I expect this function will be updated to cover the function properly. And there is high chance this piece of code will be removed before we introduce new schema. So I prefer to keep it as it is now.